tests/integration/crud/chunks/test_chunks.py

```diff
@@ -0,0 +1,164 @@
+import os
+import random
+from itertools import chain
+from pathlib import Path
+
+import numpy as np
+import pytest
+from jina.executors.indexers import BaseIndexer
+
+from jina import Document
+from jina.flow import Flow
+
+
+@pytest.fixture
+def config(tmpdir):
+    random.seed(0)
+    np.random.seed(0)
+    os.environ['JINA_TOPK_DIR'] = str(tmpdir)
+    os.environ['JINA_TOPK'] = '10'
+    yield
+    del os.environ['JINA_TOPK_DIR']
+    del os.environ['JINA_TOPK']
+
+
+d_embedding = np.random.random([9])
+c_embedding = np.random.random([9])
+
+
+def document_generator(content_same, start, num_docs, num_chunks):
```

Comment:
`content_same` is not relevant for your test, unless you plan on adding `DocIDCache` as well